lib/matplotlib/tests/test_colors.py

```diff
@@ -1135,3 +1135,16 @@ def test_hex_shorthand_notation():
 def test_DivergingNorm_deprecated():
     with pytest.warns(cbook.MatplotlibDeprecationWarning):
         norm = mcolors.DivergingNorm(vcenter=0)
+
+
+def test_repr_png():
+    cmap = plt.get_cmap('viridis')
+    png = cmap._repr_png_()
+    assert len(png) > 0
```

Comment:
I would like the test to be a bit stricter. Try loading it with PIL and check the image size.

Reply:
Good idea. Done in 9d4633874ddbf6939d75e0fd02d2a14a8da682f1.